### PR TITLE
Add test data and tests for utils.download_channel_data

### PR DESCRIPTION
**Pull request by bjdoyle**
Adds test data and tests for utils.download_channel_data.
Also makes sure that the time calculation is operating on 2 float64s - if the delta is a float32 it can cause problems.
Also added some comments and clarified others.

### CODE REVIEW

File: seerpy/utils.py [hunk 1]
```diff
@@ -124,7 +130,8 @@ def download_channel_data(data_q, download_function):
         data['segments.id'] = segments_id
         data = data[['time', 'id', 'channelGroups.id', 'segments.id'] + channel_names]
 
-        # data chunks seem to always be 10 seconds even if they don't contain that much data
+        # chunks are padded to a standard size (usually 10s) if they don't contain that much data.
```
Comment (robrkerr) [inline seerpy/utils.py:133]:
```suggestion
        # chunks are padded to a chunk size specified for the channel group (usually 10s) if they don't contain that much data.
```

File: seerpy/utils.py [hunk 2]
```diff
@@ -112,8 +117,9 @@ def download_channel_data(data_q, download_function):
 
         if meta_data['channelGroups.timestamped']:
             # data timestamp is relative to chunk start
-            chunk_start = meta_data['dataChunks.time']
-            data['time'] = data['time'] + chunk_start
+            # make sure both are float64 - sometimes mixed float arithmetic gives strange results
```
Comment (robrkerr) [inline seerpy/utils.py:120]: Yeah that makes sense given the precision we are interested.

File: tests/test_utils.py [hunk 3]
```diff
@@ -15,6 +16,134 @@
...
+            'channelGroups.samplesPerRecord': [250],
+            'channelGroups.recordsPerChunk': [10],
+            'channelGroups.compression': ['gzip'],
+            'channelGroups.signalMin': [0],
+            'channelGroups.signalMax': [0],
+            'channelGroups.exponent': [-6],
+            'channelGroups.timestamped': [True]
+        }
+        meta_data = pd.DataFrame(meta_data_dict).iloc[0]
+        print('meta_data.dtypes', meta_data.dtypes)
```
Comment (robrkerr) [inline tests/test_utils.py:47]: Did you mean to leave in these print statements?

File: seerpy/utils.py (hunk 1 above)
Comment (bjdoyle) [reply seerpy/utils.py]: I've reworded that

File: tests/test_utils.py (hunk 3 above)
Comment (bjdoyle) [reply tests/test_utils.py]: No, removed.

**Pull request merged**